### ISSUE TITLE
Teamcity Gradle 4.0 Build failed No signature of method: org.gradle.api.internal.file.collections.DefaultConfigurableFileCollection.exclude() is applicable for argument types: (java.lang.String) values: [**/*.xml]

### ISSUE THREAD

**Issue by MaadhuKumarReddy** [gradle/gradle#2810]
```
\gradlew\gradlew.bat --init-script D:\TeamCityBuildAgent\plugins\gradle-runner\scripts\init.gradle --stacktrace -PENABLE_IGNORE_FAILURES_CHECKSTYLE=true -PENABLE_IGNORE_FAILURES_FINDBUGS=true -PENABLE_SHOW_VIOLATIONS_CHECKSTYLE=true -PENABLE_FINDBUGS_CHECK=true -PENABLE_CHECKSTYLE_CHECK=true --parallel --no-daemon -Dorg.gradle.daemon=false clean build generateDeploymentManifest
[23:18:30] :	 [Step 3/4] in directory: D:\TeamCityBuildAgent\work\2b254740745f1ee0
[23:18:46] :	 [Step 3/4] Parallel execution is an incubating feature.
[23:18:46] :	 [Step 3/4] Starting Gradle in TeamCity build 18
[23:18:59]W:	 [Step 3/4] 
[23:19:00]i:	 [Step 3/4] ##teamcity[buildProblem identity='1610423665' description='groovy.lang.MissingMethodException: No signature of method: org.gradle.api.internal.file.collections.DefaultConfigurableFileCollection.exclude() is applicable for argument types: (java.lang.String) values: |[**/*.xml|]' type='gradleBuildProblem']
[23:19:01]W:	 [Step 3/4] Process exited with code 1
[23:19:01]W:	 [Step 3/4] Gradle failure report
[23:19:01]W:		 [Gradle failure report] FAILURE: Build failed with an exception.
[23:19:01]W:		 [Gradle failure report] 
[23:19:01]W:		 [Gradle failure report] * Where:
[23:19:01]W:		 [Gradle failure report] Build file 'D:\TeamCityBuildAgent\work\2b254740745f1ee0\build.gradle' line: 33
[23:19:01]W:		 [Gradle failure report] 
[23:19:01]W:		 [Gradle failure report] * What went wrong:
[23:19:01]W:		 [Gradle failure report] A problem occurred evaluating root project 'reimagine'.
[23:19:01]W:		 [Gradle failure report] > Failed to apply plugin [id 'cqBundle']
[23:19:01]W:		 [Gradle failure report]    > No signature of method: org.gradle.api.internal.file.collections.DefaultConfigurableFileCollection.exclude() is applicable for argument types: (java.lang.String) values: [**/*.xml]
[23:19:01]W:		 [Gradle failure report] 
[23:19:01]W:		 [Gradle failure report] * Try:
[23:19:01]W:		 [Gradle failure report] Run with --info or --debug option to get more log output.
[23:19:01]W:		 [Gradle failure report] 
[23:19:01]W:		 [Gradle failure report] * Exception is:
[23:19:01]W:		 [Gradle failure report] org.gradle.api.GradleScriptException: A problem occurred evaluating root project 'reimagine'.
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.groovy.scripts.internal.DefaultScriptRunnerFactory$ScriptRunnerImpl.run(DefaultScriptRunnerFactory.java:92)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.configuration.DefaultScriptPluginFactory$ScriptPluginImpl$2.run(DefaultScriptPluginFactory.java:187)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.configuration.ProjectScriptTarget.addConfiguration(ProjectScriptTarget.java:77)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.configuration.DefaultScriptPluginFactory$ScriptPluginImpl.apply(DefaultScriptPluginFactory.java:192)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.configuration.BuildOperationScriptPlugin$1.run(BuildOperationScriptPlugin.java:61)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:317)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:309)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.internal.progress.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:185)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:95)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.configuration.BuildOperationScriptPlugin.apply(BuildOperationScriptPlugin.java:58)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.configuration.project.BuildScriptProcessor.execute(BuildScriptProcessor.java:39)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.configuration.project.BuildScriptProcessor.execute(BuildScriptProcessor.java:26)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.configuration.project.ConfigureActionsProjectEvaluator.evaluate(ConfigureActionsProjectEvaluator.java:34)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.configuration.project.LifecycleProjectEvaluator.doConfigure(LifecycleProjectEvaluator.java:63)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.configuration.project.LifecycleProjectEvaluator.access$100(LifecycleProjectEvaluator.java:33)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.configuration.project.LifecycleProjectEvaluator$ConfigureProject.run(LifecycleProjectEvaluator.java:103)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:317)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:309)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.internal.progress.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:185)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:95)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.configuration.project.LifecycleProjectEvaluator.evaluate(LifecycleProjectEvaluator.java:49)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.api.internal.project.DefaultProject.evaluate(DefaultProject.java:654)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.api.internal.project.DefaultProject.evaluate(DefaultProject.java:132)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.execution.TaskPathProjectEvaluator.configure(TaskPathProjectEvaluator.java:35)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.execution.TaskPathProjectEvaluator.configureHierarchy(TaskPathProjectEvaluator.java:60)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.configuration.DefaultBuildConfigurer.configure(DefaultBuildConfigurer.java:38)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.initialization.DefaultGradleLauncher$ConfigureBuild.run(DefaultGradleLauncher.java:195)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:317)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:309)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.internal.progress.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:185)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:95)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.initialization.DefaultGradleLauncher.doBuildStages(DefaultGradleLauncher.java:157)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.initialization.DefaultGradleLauncher.access$200(DefaultGradleLauncher.java:44)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.initialization.DefaultGradleLauncher$1.run(DefaultGradleLauncher.java:122)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.internal.work.DefaultWorkerLeaseService$2.execute(DefaultWorkerLeaseService.java:124)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.initialization.DefaultGradleLauncher.doBuild(DefaultGradleLauncher.java:116)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.initialization.DefaultGradleLauncher.run(DefaultGradleLauncher.java:99)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.launcher.exec.GradleBuildController.run(GradleBuildController.java:66)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.tooling.internal.provider.ExecuteBuildActionRunner.run(ExecuteBuildActionRunner.java:28)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.launcher.exec.ChainingBuildActionRunner.run(ChainingBuildActionRunner.java:35)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.tooling.internal.provider.ValidatingBuildActionRunner.run(ValidatingBuildActionRunner.java:32)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.launcher.exec.RunAsBuildOperationBuildActionRunner$1.run(RunAsBuildOperationBuildActionRunner.java:43)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:317)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:309)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.internal.progress.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:185)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:95)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.launcher.exec.RunAsBuildOperationBuildActionRunner.run(RunAsBuildOperationBuildActionRunner.java:40)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.tooling.internal.provider.SubscribableBuildActionRunner.run(SubscribableBuildActionRunner.java:51)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:44)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:28)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.tooling.internal.provider.ContinuousBuildActionExecuter.execute(ContinuousBuildActionExecuter.java:75)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.tooling.internal.provider.ContinuousBuildActionExecuter.execute(ContinuousBuildActionExecuter.java:49)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.tooling.internal.provider.ServicesSetupBuildActionExecuter.execute(ServicesSetupBuildActionExecuter.java:51)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.tooling.internal.provider.ServicesSetupBuildActionExecuter.execute(ServicesSetupBuildActionExecuter.java:32)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.tooling.internal.provider.GradleThreadBuildActionExecuter.execute(GradleThreadBuildActionExecuter.java:36)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.tooling.internal.provider.GradleThreadBuildActionExecuter.execute(GradleThreadBuildActionExecuter.java:25)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.tooling.internal.provider.StartParamsValidatingActionExecuter.execute(StartParamsValidatingActionExecuter.java:64)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.tooling.internal.provider.StartParamsValidatingActionExecuter.execute(StartParamsValidatingActionExecuter.java:29)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.tooling.internal.provider.SessionFailureReportingActionExecuter.execute(SessionFailureReportingActionExecuter.java:55)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.tooling.internal.provider.SessionFailureReportingActionExecuter.execute(SessionFailureReportingActionExecuter.java:42)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.tooling.internal.provider.SetupLoggingActionExecuter.execute(SetupLoggingActionExecuter.java:50)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.tooling.internal.provider.SetupLoggingActionExecuter.execute(SetupLoggingActionExecuter.java:30)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.launcher.cli.RunBuildAction.run(RunBuildAction.java:51)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.internal.Actions$RunnableActionAdapter.execute(Actions.java:173)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.launcher.cli.CommandLineActionFactory$ParseAndBuildAction.execute(CommandLineActionFactory.java:287)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.launcher.cli.CommandLineActionFactory$ParseAndBuildAction.execute(CommandLineActionFactory.java:260)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.launcher.cli.JavaRuntimeValidationAction.execute(JavaRuntimeValidationAction.java:33)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.launcher.cli.JavaRuntimeValidationAction.execute(JavaRuntimeValidationAction.java:24)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.launcher.cli.ExceptionReportingAction.execute(ExceptionReportingAction.java:33)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.launcher.cli.ExceptionReportingAction.execute(ExceptionReportingAction.java:22)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.launcher.cli.CommandLineActionFactory$WithLogging.execute(CommandLineActionFactory.java:253)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.launcher.cli.CommandLineActionFactory$WithLogging.execute(CommandLineActionFactory.java:182)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.launcher.Main.doAction(Main.java:33)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.launcher.bootstrap.EntryPoint.run(EntryPoint.java:45)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.launcher.bootstrap.ProcessBootstrap.runNoExit(ProcessBootstrap.java:60)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.launcher.bootstrap.ProcessBootstrap.run(ProcessBootstrap.java:37)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.launcher.GradleMain.main(GradleMain.java:23)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.wrapper.BootstrapMainStarter.start(BootstrapMainStarter.java:30)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.wrapper.WrapperExecutor.execute(WrapperExecutor.java:129)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.wrapper.GradleWrapperMain.main(GradleWrapperMain.java:61)
[23:19:01]W:		 [Gradle failure report] Caused by: org.gradle.api.internal.plugins.PluginApplicationException: Failed to apply plugin [id 'cqBundle']
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.api.internal.plugins.DefaultPluginManager.doApply(DefaultPluginManager.java:150)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.api.internal.plugins.DefaultPluginManager.apply(DefaultPluginManager.java:125)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.api.internal.plugins.DefaultObjectConfigurationAction.applyType(DefaultObjectConfigurationAction.java:113)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.api.internal.plugins.DefaultObjectConfigurationAction.access$200(DefaultObjectConfigurationAction.java:36)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.api.internal.plugins.DefaultObjectConfigurationAction$3.run(DefaultObjectConfigurationAction.java:80)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.api.internal.plugins.DefaultObjectConfigurationAction.execute(DefaultObjectConfigurationAction.java:136)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.api.internal.project.AbstractPluginAware.apply(AbstractPluginAware.java:46)
[23:19:01]W:		 [Gradle failure report] 	at build_ehxy1jcvkvanlmf4kzq3vy8jq$_run_closure3.doCall(D:\TeamCityBuildAgent\work\2b254740745f1ee0\build.gradle:33)
[23:19:01]W:		 [Gradle failure report] 	at build_ehxy1jcvkvanlmf4kzq3vy8jq.run(D:\TeamCityBuildAgent\work\2b254740745f1ee0\build.gradle:32)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.groovy.scripts.internal.DefaultScriptRunnerFactory$ScriptRunnerImpl.run(DefaultScriptRunnerFactory.java:90)
[23:19:01]W:		 [Gradle failure report] 	... 79 more
[23:19:01]W:		 [Gradle failure report] Caused by: groovy.lang.MissingMethodException: No signature of method: org.gradle.api.internal.file.collections.DefaultConfigurableFileCollection.exclude() is applicable for argument types: (java.lang.String) values: [**/*.xml]
[23:19:01]W:		 [Gradle failure report] 	at com.intel.ccap.gradle.plugins.bundle.CqBundlePlugin$_apply_closure8.doCall(CqBundlePlugin.groovy:159)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.api.internal.ClosureBackedAction.execute(ClosureBackedAction.java:71)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.util.ConfigureUtil.configureTarget(ConfigureUtil.java:160)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.util.ConfigureUtil.configureSelf(ConfigureUtil.java:136)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.api.internal.AbstractTask.configure(AbstractTask.java:595)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.api.internal.AbstractTask.configure(AbstractTask.java:81)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.util.ConfigureUtil.configure(ConfigureUtil.java:104)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.util.ConfigureUtil$1.execute(ConfigureUtil.java:123)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.internal.Actions$FilteredAction.execute(Actions.java:205)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.api.internal.DefaultDomainObjectCollection.all(DefaultDomainObjectCollection.java:136)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.api.internal.DefaultDomainObjectCollection.all(DefaultDomainObjectCollection.java:154)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.api.internal.tasks.RealizableTaskCollection.all(RealizableTaskCollection.java:197)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.api.internal.DefaultDomainObjectCollection.withType(DefaultDomainObjectCollection.java:165)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.api.DomainObjectCollection$withType.call(Unknown Source)
[23:19:01]W:		 [Gradle failure report] 	at com.intel.ccap.gradle.plugins.bundle.CqBundlePlugin.apply(CqBundlePlugin.groovy:149)
[23:19:01]W:		 [Gradle failure report] 	at com.intel.ccap.gradle.plugins.bundle.CqBundlePlugin.apply(CqBundlePlugin.groovy)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.api.internal.plugins.ImperativeOnlyPluginTarget.applyImperative(ImperativeOnlyPluginTarget.java:41)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.api.internal.plugins.RuleBasedPluginTarget.applyImperative(RuleBasedPluginTarget.java:49)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.api.internal.plugins.DefaultPluginManager.addPlugin(DefaultPluginManager.java:165)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.api.internal.plugins.DefaultPluginManager.access$200(DefaultPluginManager.java:47)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.api.internal.plugins.DefaultPluginManager$AddPluginBuildOperation.run(DefaultPluginManager.java:252)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:317)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:309)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.internal.progress.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:185)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:95)
[23:19:01]W:		 [Gradle failure report] 	at org.gradle.api.internal.plugins.DefaultPluginManager.doApply(DefaultPluginManager.java:144)
[23:19:01]W:		 [Gradle failure report] 	... 88 more
[23:19:01]W:		 [Gradle failure report] 
[23:19:01]W:		 [Gradle failure report] 
[23:19:01]W:		 [Gradle failure report] BUILD FAILED in 29s
[23:19:12]E:	 [Step 3/4] Step Gradle Build (Gradle) failed
```

**Comment by blindpirate** [comment]
Looks like an API used by `com.intel.ccap.gradle.plugins.bundle.CqBundlePlugin` is missing (not sure public API or internal API). Which version were you using previously?

**Comment by bmuschko** [comment]
Have a look at at `com.intel.ccap.gradle.plugins.bundle.CqBundlePlugin$_apply_closure8.doCall(CqBundlePlugin.groovy:159)`. The error message states the following:

> org.gradle.api.internal.file.collections.DefaultConfigurableFileCollection.exclude() is applicable for argument types: (java.lang.String) values: [**/*.xml]

Please provide the code you are using there.

**Comment by MaadhuKumarReddy** [comment]
project.tasks.withType(FindBugs) { FindBugs task ->                                               
                task.reports.with {
                    def useHtmlReport = buildProperties.get("ENABLE_FINDBUGS_XML") as boolean
                    html.enabled = useHtmlReport
                    xml.enabled = !useHtmlReport
                }

                // Workaround for FindBugs logging issue:
                // https://discuss.gradle.org/t/findbugs-plugin-issue-throwing-zipexception/2010/5
                FileCollection findBugsClasses = task.classes
                findBugsClasses.exclude '**/*.xml'
                findBugsClasses.exclude '**/*.properties'
                task.classes = findBugsClasses
            }
        }

**Comment by MaadhuKumarReddy** [comment]
The same code is working on my local but it is failed in teamcity

**Comment by big-guy** [comment]
@MaadhuKumarReddy that workaround shouldn't be necessary.  Can you try removing it and seeing if it fixes your problem?

**Comment by bmuschko** [comment]
The code you have there is a workaround for an issue that has been fixed with Gradle 4.0 if I am not mistaken. You can likely just delete it. Please see https://github.com/gradle/gradle/issues/1094 for more information.

**Comment by MaadhuKumarReddy** [comment]
Issue got resolved after removing the workaround.
